cupy/fallback_mode/data_tranfer.py

```diff
@@ -0,0 +1,59 @@
+"""
+Data transfer methods
+"""
+
+import cupy as cp
+import numpy as np
+
+
+def vram2ram(args, kwargs):
+    """
+    Transfers ndarrays in *args, **kwargs from GPU to CPU.
+
+    Args:
+        args (tuple): Arguments.
+        kwargs (dict): Keyword arguments.
+
+    Returns:
+        cpu_args (tuple): Arguments in CPU.
+        cpu_kwargs (dict): Keyword arguments in CPU.
+    """
+    cpu_args = []
+    cpu_kwargs = {}
+
+    for arg in args:
+        if isinstance(arg, cp.ndarray):
+            cpu_args.append(cp.asnumpy(arg))
+        else:
+            cpu_args.append(arg)
+
+    for arg_name, arg in kwargs.items():
+        if isinstance(arg, cp.ndarray):
+            cpu_kwargs[arg_name] = cp.asnumpy(arg)
+        else:
+            cpu_kwargs[arg_name] = arg
+
+    return tuple(cpu_args), cpu_kwargs
+
+
+def ram2vram(res):
+    """
+    Transfers ndarrays in *args, **kwargs from CPU to GPU.
```

Comment:
This comment seems incorrect because sometimes the return value is not cupy array. How about renaming this function to `_get_cupy_args` for example?

Reply:
Okay..